backend/test/acceptance/api.controllerregistrations.spec.js

```diff
@@ -25,7 +28,6 @@ module.exports = function ({ agent, sandbox, auth, k8s }) {
     expect(res).to.have.status(200)
     expect(res).to.be.json
     expect(res.body).to.have.length(2)
-    expect(res.body[0].name).eql('foo')
-    expect(res.body[0].version).eql('v1.0.0')
+    expect(res.body[0]).to.include({ name: 'foo', version: 'v1.0.0' })
```

Comment:
I would expect
```
expect(res.body).to.include({ name: 'foo', version: 'v1.0.0' })
```
or
```
expect(res.body[0]).to.eql({ name: 'foo', version: 'v1.0.0' })
```

Reply:
`expect(res.body).to.include({ name: 'foo', version: 'v1.0.0' })`

does not work,

`expect(res.body[0]).to.eql({ name: 'foo', version: 'v1.0.0' })`

also does not work as the object has other properties.

Reply:
Ok